evap/evaluation/templates/evap_course_edit_js.html

```diff
@@ -2,48 +2,90 @@
 {% load evaluation_templatetags %}
 
 <script type="text/javascript">
-         function makeClickHandler(i, contributorCount) {
-            return function () {
-                // if the responsible checkbox is selected
-                if($("#id_contributions-" + i + "-responsible").is(":checked")) {
-                    // uncheck and enable all
-                    for(var j = 0; j < contributorCount; j++){
-                        $("#id_contributions-" + j + "-responsible").prop("checked", false);
-                        $("#id_contributions-" + j + "-can_edit").prop("disabled", false);
-                    }
+    function makeClickHandler(i, contributorCount, responsibility) {
+        return function () {
+            // don't do anything on disabled buttons
+            if ($(this).hasClass("disabled"))
+                return false;
 
-                    // check self
-                    $("#id_contributions-" + i + "-responsible").prop("checked", true);
+            // if the responsible button is selected
+            if(responsibility == "RESPONSIBLE") {
+                // uncheck all buttons
+                uncheckButtons("input[name=contributions-" + i + "-comment_visibility]");
+                uncheckButtons("input[name=contributions-" + i + "-responsibility]");
 
-                    // check and disable can-edit
-                    $("#id_contributions-" + i + "-can_edit").prop("checked", true);
-                    $("#id_contributions-" + i + "-can_edit").prop("disabled", true);
+                // check "all comments" and disable comment buttons
+                checkButtons("input[name=contributions-" + i + "-comment_visibility][value=ALL]");
+                disableButtons("input[name=contributions-" + i + "-comment_visibility]");
+
+                // disable all other responsible buttons
+                for(var j = 0; j < contributorCount; j++){
+                    if (j != i)
+                        disableButtons("input[name=contributions-" + j + "-responsibility][value=RESPONSIBLE]");
```

Comment:
when selecting a responsible button, all other responsible buttons must be disabled, because it should only be possible to select one contributor as responsible.